wsdl/ver10/advancedsecurity/wsdl/advancedsecurity.wsdl

```diff
@@ -839,6 +875,127 @@
+						</xs:annotation>
+					</xs:enumeration>
+					<xs:enumeration value="client_secret_jwt">
+						<xs:annotation>
+							<xs:documentation>Use a HMAC signed JWT using client_secret as shared secret par OpenID Connect Core</xs:documentation>
+						</xs:annotation>
+					</xs:enumeration>
+					<xs:enumeration value="private_key_jwt">
+						<xs:annotation>
+							<xs:documentation>Use PKI signed JWT using private key par OpenID Connect Core</xs:documentation>
+						</xs:annotation>
+					</xs:enumeration>
+					<xs:enumeration value="tls_client_auth">
+						<xs:annotation>
+							<xs:documentation>Use PKI certificate to authenticate par RFC 8705</xs:documentation>
+						</xs:annotation>
+					</xs:enumeration>
+					<xs:enumeration value="self_signed_tls_client_auth">
+						<xs:annotation>
+							<xs:documentation>Use self-signed certificate to authenticate par RFC 8705</xs:documentation>
+						</xs:annotation>
+					</xs:enumeration>
+				</xs:restriction>
+			</xs:simpleType>
+			<!--===============================-->
+			<xs:complexType name="AuthorizationServerConfigurationData">
+				<xs:sequence>
+					<xs:element name="ServerUri" type="xs:anyURI">
+						<xs:annotation>
+							<xs:documentation>Authorization server address</xs:documentation>
+						</xs:annotation>
+					</xs:element>
+					<xs:element name="ClientId" type="xs:string" minOccurs="0">
+						<xs:annotation>
+							<xs:documentation>Client identifier issued by the authorzation server</xs:documentation>
+						</xs:annotation>
+					</xs:element>
+					<xs:element name="ClientSecret" type="xs:string" minOccurs="0">
+						<xs:annotation>
+								<xs:documentation>Client secret used to authenticate with the authorization server</xs:documentation>
+						</xs:annotation>
+					</xs:element>
+					<xs:element name="Scope" type="xs:string" minOccurs="0">
+						<xs:annotation>
+							<xs:documentation>The requested access scope(s)</xs:documentation>
+						</xs:annotation>
+					</xs:element>
+					<xs:any namespace="##any" processContents="lax" minOccurs="0" maxOccurs="unbounded"/>
+				</xs:sequence>
+				<xs:attribute name="Type" type="xs:string" use="required">
+					<xs:annotation>
+						<xs:documentation>The type of configuration, tas:AuthorizationServerConfigurationType lists the acceptable values</xs:documentation>
+					</xs:annotation>
+				</xs:attribute>
+				<xs:attribute name="ClientAuth" type="xs:string">
+					<xs:annotation>
+						<xs:documentation>How to authenticate with the server, tas:ClientAuthenticationMethod lists the acceptable values</xs:documentation>
+					</xs:annotation>
+				</xs:attribute>
+				<xs:anyAttribute processContents="lax"/>
+			</xs:complexType>
+			<!--===============================-->
+			<xs:complexType name="AuthorizationServerConfiguration">
+				<xs:complexContent>
+					<xs:extension base="tt:DeviceEntity">
+						<xs:sequence>
```

Comment:
I do not see Token as part of AuthorizationServerConfiguration.

Reply:
It's part of DeviceEntity which it inherits from.

Reply:
Thanks Fredrik for the clarification, I got it.